dist/docsite/figma/figma.json

```diff
@@ -5158,6 +5364,182 @@
+          "min": "1.92rem",
+          "ideal": "0.64cqw + 1.72rem",
+          "max": "2.5rem"
+        },
+        "$type": "clamp",
+        "filePath": "tokens/web/types.json5",
+        "original": {
+          "$value": {
+            "min": "1.92rem",
+            "ideal": "0.64cqw + 1.72rem",
+            "max": "2.5rem"
+          },
+          "$type": "clamp"
+        }
+      },
+      "2": {
+        "$value": {
+          "min": "2.3rem",
+          "ideal": "0.9cqw + 2.02rem",
+          "max": "3.13rem"
+        },
+        "$type": "clamp",
+        "filePath": "tokens/web/types.json5",
+        "original": {
+          "$value": {
+            "min": "2.3rem",
+            "ideal": "0.9cqw + 2.02rem",
+            "max": "3.13rem"
+          },
+          "$type": "clamp"
+        }
+      },
+      "3": {
+        "$value": {
+          "min": "2.76rem",
+          "ideal": "1.25cqw + 2.36rem",
+          "max": "3.91rem"
+        },
+        "$type": "clamp",
+        "filePath": "tokens/web/types.json5",
+        "original": {
+          "$value": {
+            "min": "2.76rem",
+            "ideal": "1.25cqw + 2.36rem",
+            "max": "3.91rem"
+          },
+          "$type": "clamp"
+        }
+      },
+      "4": {
+        "$value": {
+          "min": "3.32rem",
+          "ideal": "1.72cqw + 2.77rem",
+          "max": "4.88rem"
+        },
+        "$type": "clamp",
+        "filePath": "tokens/web/types.json5",
+        "original": {
+          "$value": {
+            "min": "3.32rem",
+            "ideal": "1.72cqw + 2.77rem",
+            "max": "4.88rem"
+          },
+          "$type": "clamp"
+        }
+      },
+      "5": {
+        "$value": {
+          "min": "3.98rem",
+          "ideal": "2.33cqw + 3.24rem",
+          "max": "6.1rem"
+        },
+        "$type": "clamp",
+        "filePath": "tokens/web/types.json5",
+        "original": {
+          "$value": {
+            "min": "3.98rem",
+            "ideal": "2.33cqw + 3.24rem",
+            "max": "6.1rem"
+          },
+          "$type": "clamp"
+        }
+      },
+      "6": {
+        "$value": {
+          "min": "4.78rem",
+          "ideal": "3.13cqw + 3.78rem",
+          "max": "7.63rem"
+        },
+        "$type": "clamp",
+        "filePath": "tokens/web/types.json5",
+        "original": {
+          "$value": {
+            "min": "4.78rem",
+            "ideal": "3.13cqw + 3.78rem",
+            "max": "7.63rem"
+          },
+          "$type": "clamp"
+        }
+      },
+      "7": {
+        "$value": {
+          "min": "5.73rem",
+          "ideal": "4.17cqw + 4.4rem",
+          "max": "9.54rem"
+        },
+        "$type": "clamp",
+        "filePath": "tokens/web/types.json5",
+        "original": {
+          "$value": {
+            "min": "5.73rem",
+            "ideal": "4.17cqw + 4.4rem",
+            "max": "9.54rem"
+          },
+          "$type": "clamp"
+        }
+      },
+      "minus": {
```

Comment:
If we change this to minus we will have a potential breaking change correct?

Reply:
we can make this change and support these legacy variants in the deprecated output - this will provide support and let us migrate without a breaking change